Exdi/exdigdbsrv/GdbSrvControllerLib/AsynchronousGdbSrvController.cpp

```diff
@@ -702,6 +702,13 @@ void AsynchronousGdbSrvController::HandleStopReply(_In_ const std::string reply,
         *pEventNotification = true;
         *pProcessorNumber = GdbSrvController::GetLastKnownActiveCpu();
     }
+    // Is it an "OK" response w/o any other field (e.g. OpenOCD can send this after 's'/'g')?
+    else if (stopReply.status.isCoreRunning)

```

Comment:
This is another change unrelated to the rename. This should be applied separately

Reply:
Ok, I'll add it separately, I have this pending in my sand box related to Ampere folks changes.